tools/proofers/PPage.inc

```diff
@@ -87,6 +87,17 @@ class PPage
         return "$projects_url/{$this->lpage->projectid}/{$this->lpage->imagefile}";
     }
 
+    function url_for_display_image($escape_amp=FALSE)
+    {
+        global $code_url;
+        if($escape_amp) $amp = '&amp;';
```

Comment:
I would prefer the conditional statement on a new line. like
`if($escape_amp)
    $amp = '&amp;';`
or even
`if($escape_amp)
{
    $amp = '&amp;';
}`

Reply:
Sure I can make that change. I was just trying to be consistent with line 116. I see another open PR to autoformat PHP. That will help here :).

Reply:
Consistency with the rest of the file is probably the right move here. As-is getting rid of escaping in the function as well (but again, not in this PR).